### PR TITLE
[WIP] AudioDSP skin and dialog changes

### PR DESCRIPTION
**Pull request by AchimTuran**
This PR sets equal places for AudioDSP during video- and music-playback.

Furthermore it moves volume and delay settings back to the OSD for subtitles- and audio-settings.

### CODE REVIEW

File: addons/skin.estuary/1080i/DialogExtendedProgressBar.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <window>
+	<defaultcontrol />
```
Comment (phil65) [inline addons/skin.estuary/1080i/DialogExtendedProgressBar.xml]: does an empty defaultcontrol help for anything?

File: addons/skin.estuary/1080i/DialogExtendedProgressBar.xml [hunk 2]
```diff
@@ -79,4 +82,4 @@
 			</control>
 		</control>
 	</controls>
-</window>
+</window>
```
Comment (phil65) [inline addons/skin.estuary/1080i/DialogExtendedProgressBar.xml]: removed newline at EOF.

**Comment by phil65** [comment]
What is all that stuff added to VideoFullScreen.xml used for?

**Comment by AchimTuran** [comment]
> What is all that stuff added to VideoFullScreen.xml used for?

I guess @HitcherUK should know this. It is a fix from him to keep the aspect ratio of the progress circle.

**Comment by HitcherUK** [comment]
I only touched the progress images unless that codec stuff was part of Estuary back when I fixed them and Achim is using that XML.

**Comment by AchimTuran** [comment]
@Hitcher UK
I hope that I didn't mess up something. I just copy pasted all from our Slack conversation.
Could you tell me what is really needed. Then I will remove the unnecessary stuff.

File: addons/skin.estuary/1080i/DialogExtendedProgressBar.xml (hunk 1 above)
Comment (HitcherUK) [inline addons/skin.estuary/1080i/DialogExtendedProgressBar.xml]: Remove

**Comment by phil65** [comment]
Any fixes / changes to progress circle shouldnt be hidden in this PR, please create a new one instead.

**Comment by AchimTuran** [comment]
> Any fixes / changes to progress circle shouldnt be hidden in this PR, please create a new one instead.

Ok, I will create a new PR.

**Comment by Rechi** [comment]
@AchimTuran this needs a rebase

**Comment by AchimTuran** [comment]
This PR is osbolete, which is the reason why I close it.

**Pull request closed without merging**